.github/workflows/release.yml

```diff
@@ -14,8 +14,6 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     permissions:
-      contents: write
-      id-token: write
```

Comment:
I think these permissions should be retained so that the package is published with provenance. See here for the line that enables this within the changesets step: https://github.com/api3dao/ois/blob/5fe6e4d3e665099b497f3e30e66abf90ed4881f9/.github/workflows/publish.yml#L53

Reply:
I have removed it because I was not sure if this repo will be private one or not in the future. If it is going to be a public one then no problem. And re-enabled the provenance setting